src/test/java/org/symphonyoss/symphony/messageml/elements/form/PersonSelectorTest.java

```diff
@@ -109,12 +109,12 @@ public void sendValidPersonSelectorWithSelectedUsersIdsMultiple() throws Excepti
   }
 
   @Test
-  public void sendInvalidPersonSelectorWithSelectedUsersIds() throws Exception {
+  public void sendInvalidPersonSelectorWithValueNotALong() throws Exception {
     expectedException.expect(InvalidInputException.class);
-    expectedException.expectMessage("Attribute \"selected-users-ids\" contains an unsupported format, only user ids separated by comma are supported");
+    expectedException.expectMessage("Attribute \"value\" contains an unsupported format, should be an array of user ids");
     context.parseMessageML("<messageML>" +
             "  <form id=\"all-elements\">" +
-            "    <person-selector name=\"person-selector\" placeholder=\"Type a person's name\" selected-users-ids=\"123,abc\"/>" +
+            "    <person-selector name=\"person-selector\" placeholder=\"Type a person's name\" value='[123,abc]'/>" +
```

Comment:
nice!!
you can also test invalid value like value="xxyz"

Reply:
sure, added!